src/plots/domain.js

```diff
@@ -90,6 +91,7 @@ exports.attributes = function(opts, extra) {
         out.column = {
             valType: 'integer',
             min: 0,
+            dflt: 0,
```

Comment:
Great. Can we :lock: this down in a jasmine test?

Reply:
good call -> https://github.com/plotly/plotly.js/pull/2399/commits/794669bf39accfc5bf3faa394c6c1c0d5d9a6e2e